docs/source/user-guide/vision/using_precomputed_predictions_vision.rst

```diff
@@ -0,0 +1,52 @@
+.. _using_precomputed_predictions:
+
+=============================
+Using Pre-computed Predictions
+=============================
+Some checks, mainly the ones related to model evaluation, require model predictions in order to run.
+In deepchecks, predictions are passed to the suite / check run method in one of the following ways:
+
+* Implementing an infer_on_batch methods in the :doc:`VisionData object </user-guide/vision/data-classes/VisionData>`,
+  that allows the checks to compute the predictions.
+* Passing the pre-computed predictions as a parameter to the check's run
+
+Passing pre-computed predictions is a simple alternative to using a model in infer_on_batch.
+It is specifically recommended to use this option if your model object is unavailable locally (for example if placed on
+a separate prediction server) or if the predicting process is computationally expensive or time consuming.
+
+The predictions should be passed to the train_predictions, test_predictions, or both arguments of the suite/check's run
+method in the appropriate format.
+
+Pre-computed Predictions Format
+-------------------------------
+The expected format is a dictionary of {sample index (int): sample predictions (tensor or list of tensors)}
```

Comment:
@shir22 1. You refer to the font and not the content? How do I make it look like your suggestion?
2. From what I understood, there is no specific format it can be probas, bboxes, etc. (JKL who implemented this part is on vacation)